Assets/MixedRealityToolkit-SDK/Features/Utilities/Solvers/SolverHandler.cs

```diff
@@ -190,10 +198,37 @@ protected virtual void AttachToNewTrackedObject()
                 case TrackedObjectType.Head:
                     TrackTransform(CameraCache.Main.transform);
                     break;
-                // Other cases will come online as ControllerFinder is ported appropriately.
+                case TrackedObjectType.LeftController:
+                    var leftController = GetVisualizerForHand(Handedness.Left);
+                    if (leftController != null)
+                    {
+                        TrackTransform(leftController.gameObject.transform);
+                    }
+                    break;
+                case TrackedObjectType.RightController:
+                    var rightController = GetVisualizerForHand(Handedness.Right);
+                    if (rightController != null)
+                    {
+                        TrackTransform(rightController.gameObject.transform);
+                    }
+                    break;
+
+                    // Other cases will come online as ControllerFinder is ported appropriately.
             }
         }
 
+        private MixedRealityControllerVisualizer GetVisualizerForHand(Handedness hand)
+        {
+            foreach (MixedRealityControllerVisualizer controller in MixedRealityManager.Instance.MixedRealitySceneObjects)
```

Comment:
This doubles down on the current requirement that model overrides must have a pose synchronizer on them.

This has already been given as feedback that it adds an unnecessary requirement (as in, there are ways to do this without adding a script; see: HTK) to adding arbitrary prefabs as overrides, as well as likely being incompatible once we load platform controller models and create GameObjects for them at runtime, since the core has no knowledge of the concrete pose synchronizers.